core/record/recordExpressionParser.js

```diff
@@ -74,7 +80,26 @@ const bindNode = (survey, record, node) => {
         : NodeDef.isBoolean(nodeDef)
           ? value === 'true'
           : value
-    }
+    },
+
+    getReachableNodeValue: nodeName => {
+      const _getNodeByGivenName = R.pipe(
```

Comment:
I could lookup the target node by name but there's still a need to verify that the node is in the set of permissible nodes.
I don't think it would make life easier to do it that way, but I could be wrong.